docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1272,6 +1272,33 @@ FROM example.testdb."customer_orders$snapshots"
 ORDER BY committed_at DESC
 ```
 
+(iceberg-create-or-replace)=
+
+#### Replacing tables
+
+The connector supports replacing a table as an atomic operation. Atomic table
+replacement creates a new snapshot with the results of the SELECT query, but
```

Comment:
What SELECT query? That only applies for CTAS .. but the connector should support both .. right? 

We have to reword this accordingly. And we probably should link to the `create-table` and `create-table-as` pages